### PR TITLE
cocc: update election procedure

### PR DESCRIPTION
**Pull request by palnabarun**
Till date, CoCC elections have been happening through CIVS. We want to
move to use https://elections.k8s.io/ based on Elekto for all voting
requirements. This would also improve the transparency of the process.

/assign @kubernetes/steering-committee 
/cc @kubernetes/code-of-conduct-committee 

/kind documentation
/priority critical-urgent

### CODE REVIEW

**Comment by MadhavJivrajani** [comment]
/hold
to prevent accidental merge in case of lgtm's coming in.

**Review by mrbobbytables — COMMENTED** [review]
lgtm with steering hat
looping in CoCC manually
/assign @detiber @endocrimes @hlipsig @jeremyrickard @salaxander

**Review by cblecker — APPROVED** [review]

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by salaxander — APPROVED** [review]

**Comment by endocrimes** [comment]
"lgtm but" - for easy future reference are there docs on how elekto differs from the previous method anywhere?

**Comment by mrbobbytables** [comment]
@endocrimes for docs on elekto: https://elekto.dev
Before it was the standard CIVS / email ballot. Elekto uses the same condorcet voting system, just a lot easier for us to manage as a whole.

File: committee-code-of-conduct/election.md [hunk 1]
```diff
@@ -28,10 +28,9 @@ This document outlines the process, for Code of Conduct Committee (CoCC) electio
 The steering committee will announce a request for nominations some time in July from anyone meeting the [eligibility for candidacy](#eligibility-for-candidacy) guidelines.
 
 The steering committee will accept nominations for at least 1 week. Though the steering
-committee may choose to accept nominations for longer.  After the announced nomination
-period the steering committee will hold a private ballot election using time-limited
-[Condorcet] ranking use the [Schulze method] to break ties. The steering committee
-will publicly announce the results of the election.
+committee may choose to accept nominations for longer. Once the announced nomination
+period ends, steering committee will hold an election using [Elekto](https://elections.k8s.io/)
```
Comment (BenTheElder) [inline committee-code-of-conduct/election.md:32]: I wonder if we should dedupe this to more generally document our current elections implementation, but that's not important at the moment.

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

File: committee-code-of-conduct/election.md (hunk 1 above)
Comment (palnabarun) [reply committee-code-of-conduct/election.md:32]: I agree we can have a relook at the election doc to de-dup content at some places. We can take it as a follow-up.

**Comment by palnabarun** [comment]
We have LGTMs from 4 out of 5 CoCC members and 5 out of 7 Steering Committee members, in addition to me, who proposed the changes.

/hold cancel

**Pull request merged**